backends/llamacpp/make-gguf.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+[ "$#" -ge 2 ] || {
+	echo "Usage: $0 <GGUF> <MODEL_ID> [<REV>]" >&2
+	return 1
+}
+
+GGUF="$1"
+GGUF_DIR=$(dirname "$GGUF")
+GGUF_TMP="model.src/tmp.gguf"
+MODEL_ID="$2"
+MODEL_DIR="model.src/$2"
+REV="${3-main}"
+
```

Comment:
```suggestion
HF_HUB_ENABLE_HF_TRANSFER=1
```

Reply:
It is derived from the env of the docker